pkg/apps/blackduck/latest/deployer.go

```diff
@@ -110,15 +110,25 @@ func (hc *Creater) GetComponents(blackduck *blackduckapi.Blackduck) (*api.Compon
 	componentList.ConfigMaps = append(componentList.ConfigMaps, containerCreater.GetConfigmaps()...)
 
 	//Secrets
-	// nginx certificatea
+	// nginx certificate
 	cert, key, _ := hc.getTLSCertKeyOrCreate(blackduck)
 	if !hc.config.DryRun {
 		secret, err := util.GetSecret(hc.kubeClient, hc.config.Namespace, "blackduck-secret")
 		if err != nil {
 			log.Errorf("unable to find Synopsys Operator blackduck-secret in %s namespace due to %+v", hc.config.Namespace, err)
 			return nil, err
 		}
-		componentList.Secrets = append(componentList.Secrets, containerCreater.GetSecrets(cert, key, secret.Data["SEAL_KEY"])...)
+
+		// if Black Duck instance level Seal Key is provided, then use it else use the operator level seal key
+		sealKey := secret.Data["SEAL_KEY"]
```

Comment:
Is there a format specification for a seal key?  I see some verification in the `update` command for the length, but none in `create` or on the operator side.  If there is a particular specification, can we please provide an appropriate concise description as well as add an example?

Reply:
added validations